data/json/monsters/monsters.json

```diff
@@ -1116,6 +1116,67 @@
     "death_function": [ "BROKEN" ],
     "flags": [ "SEES", "ELECTRONIC", "NO_BREATHE", "FIREPROOF", "PUSH_MON", "HEARS", "PACIFIST", "GRABS", "INTERIOR_AMMO" ]
   },
+  {
+    "id": "mon_grocerybot",
+    "type": "MONSTER",
+    "name": "grocery bot",
+    "description": "A popular product from Uncanny, this friendly android can carry your bag or push your Grocery Cart for you.  You know it's friendly because its human face is constantly smiling at you.  According to the ads it uses a state-machine of a complexity never seen before, Uncanny even hired professional actors to record its lines.  Examine to befriend.",
+    "default_faction": "mech_bot",
+    "species": [ "ROBOT" ],
+    "volume": "62500 ml",
+    "weight": 81500,
+    "hp": 80,
+    "speed": 100,
+    "material": [ "steel" ],
+    "symbol": "g",
+    "color": "white",
+    "bodytype": "human",
+    "morale": 100,
+    "dodge": 3,
+    "armor_bash": 10,
+    "armor_cut": 10,
+    "revert_to_itype": "bot_grocerybot",
+    "special_attacks": [ [ "PAID_BOT", 1 ] ],
+    "death_drops": { "groups": [ [ "robots", 4 ] ] },
+    "death_function": [ "BROKEN" ],
+    "flags": [
+      "SEES",
+      "ELECTRONIC",
+      "NO_BREATHE",
+      "FIREPROOF",
+      "PUSH_MON",
+      "HEARS",
+      "PACIFIST",
+      "PAY_BOT",
+      "PET_HARNESSABLE",
+      "PATH_AVOID_DANGER_2"
+    ]
+  },
+  {
+    "id": "mon_grocerybot_busted",
+    "type": "MONSTER",
+    "name": "busted grocery bot",
+    "description": "A busted grocery bot, it can still carry your bag but its arms are to damaged to push a Grocery Cart.  Despite its current state it's still smiling at you.  Examine to befriend.",
```

Comment:
```suggestion
    "description": "A busted grocery bot, it can still carry your bag but its arms are too damaged to push a Grocery Cart.  Despite its current state it's still smiling at you.  Examine to befriend.",
```